src/containers/Application/Part0.js

```diff
@@ -1,6 +1,6 @@
 import React, { useState, useEffect } from 'react'
 import { useLocation } from 'wouter'
-import VaccineInfo from '../../components/ApplicationForm/VaccineInfo'
+import HackathonInfo from '../../components/ApplicationForm/HackathonInfo'
```

Comment:
Since there is nothing about vaccination on the first page of the application anymore